### DISCUSSION TITLE
is it possible to implement on virtual machines

### DISCUSSION THREAD

**Discussion by kavyashazab** [dec0dOS/zero-ui#68] [General]
hello sir,
actually i tried to implement on virtual machines but facing some (network) issues while logging in so i have implemented on GCP.Is there any possibility to implement on virtual machines.

**Comment by kavyashazab** [comment]
hello sir,
can we get the zero installation package for ubuntu to run on virtual machines

**Reply by dec0dOS** [reply]
Hello, @kavyashazab
Using the docker installation does not prevent you from using the ZeroUI on the virtual machine.

**Reply by kavyashazab** [reply]
Hello sir,
Instead of using docker container I would like to implement it on vm directly through web server.is it possible?

**Reply by dec0dOS** [reply]
No, please follow the installation guide. You could try to install the ZeroUI with another method, but it wouldn't be officially supported.
